Assets/Shopify/UIToolkit/Components/RemoteImageLoader.cs

```diff
@@ -0,0 +1,77 @@
+namespace Shopify.UIToolkit {
+    using UnityEngine;
+    using UnityEngine.UI;
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// A behaviour that fetches a remote image (JPEG/PNG) from the web.
+    /// </summary>
+    [RequireComponent(typeof(Image))]
+    public class RemoteImageLoader : MonoBehaviour {
+
+        public string ImageURL;
+
+        private Image _image;
+
+        private delegate void RemoteImageCompletionDelegate(Texture2D texture, string error);
+
+        void Start() {
+            _image = gameObject.GetComponent<Image>();
+        } 
+
+        public void LoadImage() {
```

Comment:
Should have docs here

Reply:
Also why not pass the URL in here? Then you wont have a useless field in the editor for the url (if you know what the image is gonna be ahead of time then you might as well download it and get mipmaps and other nice things)